components/wifi.rst

```diff
@@ -7,8 +7,9 @@ WiFi Component
     :keywords: WiFi, WLAN, ESP8266, ESP32
 
 This core ESPHome component sets up WiFi connections to access points
-for you. It needs to be in your configuration or otherwise ESPHome
-will fail in the config validation stage.
+for you. You need to have a network configuration in your sketch (either Wifi ou Ethernet)
```

Comment:
the term `sketch` is not really common for yaml configuration afaik. Maybe just leave "in your sketch" out of the text? Also one little typo "Wifi ou Ethernet" (should be "or")